src/orderbook.js

```diff
@@ -828,6 +828,32 @@ OrderBook.prototype.isBalanceChangeNode = function(node) {
   return true;
 };
 
+OrderBook.prototype._canRunAutobridgeCalc = function(): boolean {
+  return !this._calculatorRunning && (Date.now() - this._lastCalcDuration >= 100);
```

Comment:
>  Shouldn't the lock be sufficient?

I don't know, I haven't tested these extreme conditions. I wrote this in hope that it will give user's browser some chances in case that device is slow and load is high. It won't matter in normal conditions.